src/catalog/catalog-menu.md

```diff
@@ -4,44 +4,48 @@ redirect_from:
   - /catalog.html
 ---
 
-The Catalog Menu provides easy access to product creation, category and inventory management tools{% if "Default.EE Only,Default.CE Only" contains site.edition %}.{% endif %} {% if "Default.B2B Only" contains site.edition %}, as well as shared catalogs for custom pricing.{% endif %}
+The Catalog Menu provides easy access to product creation, category, and inventory management tools{% if "Default.EE Only,Default.CE Only" contains site.edition %}.{% endif %} {% if "Default.B2B Only" contains site.edition %}, as well as shared catalogs for custom pricing.{% endif %}
 
 <!--{% if "Default.CE Only" contains site.edition %}-->
 ![]({{ site.baseurl }}{% link images/images/admin-menu-catalog.png %}){: .zoom}
-*Catalog Menu*
+_Catalog Menu_
 <!--{% endif %}-->
 <!--{% if "Default.EE Only" contains site.edition %}-->
 ![]({{ site.baseurl }}{% link images/images-ee/admin-menu-catalog-ee.png %}){: .zoom}
-*Catalog Menu*
+_Catalog Menu_
 <!--{% endif %}-->
 <!--{% if "Default.B2B Only" contains site.edition %}-->
 ![]({{ site.baseurl }}{% link images/images-b2b/admin-menu-catalog.png %}){: .zoom}
-*Catalog Menu*
+_Catalog Menu_
 <!--{% endif %}-->
 
-## To display the Catalog menu
+## Display the Catalog menu
 
-On the Admin sidebar, tap <span class="btn"> Catalog </span>.
+On the Admin sidebar, click **Catalog**.
 
 ![]({{ site.baseurl }}{% link images/images/products-grid.png %}){: .zoom}
-[*Catalog products grid* ]({{ site.baseurl }}{% link catalog/products.md %})
+[_Catalog products grid_]({{ site.baseurl }}{% link catalog/products.md %})
 
-## [Products]({{ site.baseurl }}{% link catalog/products.md %})
+## Products
```

Comment:
I'm also noticing that headings are worded in diff ways. How do we want to address consistency here? Or does it matter?

<img width="828" alt="Screen Shot 2019-09-16 at 7 38 20 AM" src="https://user-images.githubusercontent.com/14031342/64958547-11904c80-d855-11e9-90f7-63b344b5032d.png">

Reply:
Lots of inconsistencies we want to look at and address going forward.